src/dataframerow/dataframerow.jl

```diff
@@ -63,7 +63,11 @@ Base.hash(r::DataFrameRow, h::UInt = zero(UInt)) =
 # only the rows of the same DataFrame could be compared
 # rows are equal if they have the same values (while the row indices could differ)
 # if all non-missing values are equal, but there are missings, returns missing
-Base.:(==)(r1::DataFrameRow, r2::DataFrameRow) = isequal(r1, r2)
+Base.:(==)(r1::DataFrameRow, r2::DataFrameRow) = isequivalent(r1, r2)
```

Comment:
@rofinn I'm not sure what you mean here? Do you mean just not making this change here? Or something else?

Reply:
I guess so, but it seems weird to me that `isequal` is being used rather than `==` though. I guess if we can support different comparison operators then we could have the default be `==` because that is the least confusing and allow people to change that behaviour for specific operations.